test/gen_rmq/consumer/queue_configuration_test.exs

```diff
@@ -0,0 +1,60 @@
+defmodule GenRMQ.Consumer.QueueConfigurationTest do
```

Comment:
I would prefer to use asserts. Just for consistency with other test modules.

Reply:
Gotcha.  When I get a chance I'll add the assert based tests.